tests/publiccloud/run_ltp.pm

```diff
@@ -26,6 +26,17 @@ use version_utils;
 my $kirk_virtualenv = 'kirk-virtualenv';
 our $root_dir = '/root';
 
+sub prepare_instance {
```

Comment:
can we limit this PR to just changes which are related to stop checking for guest register  service during instance creation ? you doing some extra code optimisation might or might not be good idea but it certainly unrelated to goal which we trying to achieve